src/client/firefox/create.js

```diff
@@ -54,6 +54,7 @@ export function createSource(
     relativeUrl: source.url,
     isPrettyPrinted: false,
     isWasm: false,
+    isExtension: source.url && source.url.startsWith("moz-extension://"),
```

Comment:
Does this address the extensions from Chrome browser?

Reply:
No, I've just fixes that - fixed regexp and moved this set to common source action creator